src/onUnhandledRequest.ts

```diff
@@ -17,7 +17,17 @@ export function onUnhandledRequest(
     return
   }
 
-  const message = `captured a ${request.method} ${request.url} request without a corresponding request handler.`
+  const message = `captured a ${request.method} ${
+    request.url
+  } request without a corresponding request handler.
+
+  If you wish to intercept this request, consider creating a request handler for it:
+
+  rest.${request.method.toLowerCase()}("${
+    request.url.pathname
```

Comment:
As you may be drawing inspiration from here:

https://github.com/mswjs/msw/blob/a126de22c7604f84e1a904c3025282ad25ebb00d/src/rest.ts#L100

I notice that relying on `pathname` won't work in case of absolute URL given to a request handler.

- Given `https://test.mswjs.io/user`
- Suggestion: '/user'

That will not work during request capturing. I think we need to distinguish between absolute and relative URLs, providing the proper request handler URL here. Let's brainstorm on this, if you don't mind.

Reply:
At a quick pass, the code that you have in `rest.ts` will solve this if I'm understanding the intended behavior correctly. Quick example:

```
const isRelativeRequest = request.referrer.startsWith(request.url.origin)
  const publicUrl = isRelativeRequest
    ? request.url.pathname
    : format({
        protocol: request.url.protocol,
        host: request.url.host,
        pathname: request.url.pathname,
      })

  const message = `captured a ${request.method} ${
    request.url
  } request without a corresponding request handler.

  If you wish to intercept this request, consider creating a request handler for it:

  rest.${request.method.toLowerCase()}('${publicUrl}', (req, res, ctx) => {
    return res(ctx.text('body'))
  })`

In usage: 

```
useEffect(() => {
fetch("/this-is-a-bad-request", {
      method: "POST",
      body: JSON.stringify({ data: "two" }),
    });

    fetch("https://superbatmandudeguy.com/this-is-a-bad-request");
}, [])
```

Console output:

![image](https://user-images.githubusercontent.com/784953/90345585-b7ed5080-dfd6-11ea-8088-1d253ef15514.png)

Reply:
I pushed a quick update to show this, but we should probably add a couple more test cases to show both the absolute and relative behaviors, as well as add a test for the new util if you want to keep it. I think this achieves the goal, but I'm not sure if its exactly right. The previous behavior on the absolute URL on the `setupServer` test returned a trailing slash, and the updated test does this as well... is this correct behavior?